docs/kr.tree

```diff
@@ -276,6 +275,7 @@
             <toc-element id="set-operations.md" toc-title="Set-specific operations"/>
             <toc-element id="map-operations.md" toc-title="Map-specific operations"/>
         </toc-element>
+        <toc-element id="sequences.md"/>
```

Comment:
Is it right to move this page out of collections?

Reply:
No, I don't see it as an improvement. Sequences are very close to collections semantically.

Reply:
Ok, fixed. Check the text, please.